packages/realm-server/migrations/1726475945119_add-resource-created-at-to-boxel-index.js

```diff
@@ -0,0 +1,12 @@
+exports.up = (pgm) => {
+  pgm.addColumns('boxel_index', {
+    resource_created_at: { type: 'bigint' },
```

Comment:
I didn't want to name this simply `created_at` since that could be confusing - one could think this is the time of index entry insert. This makes it clearer to see this creation time is the resource the index entry represents (instance, module)